### ISSUE TITLE
Saving a PathFollow2D node as a new scene creates a Timer node in the new scene

### ISSUE THREAD

**Issue by InfiniteCactusDev** [godotengine/godot#92970]
### Tested versions

4.2.1.stable.mono

### System information

Windows 11

### Issue description

When you "Save Branch as Scene" on a PathFollow2D node, the new scene has a Timer node as a child. This is unexpected behavior.

### Steps to reproduce

Create a PathFollow2D node.
Right click and press "Save Branch as Scene".
Open the newly created scene.
Note it contains a Timer node as child.

![image](https://github.com/godotengine/godot/assets/145286353/aacb4956-8754-4cb8-8d9f-da387c21200f)

![image](https://github.com/godotengine/godot/assets/145286353/a04659a1-c41d-4a3c-aadc-6b885594fe46)

### Minimal reproduction project (MRP)

Not needed.

**Comment by AThousandShips** [comment]
Please try with 4.2.2 to make sure this still occurs for you and hasn't already been fixed

**Comment by InfiniteCactusDev** [comment]
That was quick! It indeed seems to have been fixed in 4.3.beta1

**Comment by AThousandShips** [comment]
It has? That's odd, it should still be present, are you sure it's fixed? The code hasn't been changed

**Comment by InfiniteCactusDev** [comment]
Well, I was not able to reproduce this in 4.3.beta1.. Let me try again

**Comment by AThousandShips** [comment]
Might be that this specific behaviour has been fixed and that other cases might still occur, such as duplicating the path follow directly, but if you encounter this again you can reopen this, otherwise let's consider it fixed!
